engine/src/main/java/io/camunda/zeebe/engine/processing/bpmn/behavior/BpmnEventPublicationBehavior.java

```diff
@@ -79,4 +86,50 @@ public Either<Failure, CatchEventTuple> findErrorCatchEvent(
     final var flowScopeInstance = elementInstanceState.getInstance(context.getFlowScopeKey());
     return catchEventAnalyzer.findCatchEvent(errorCode, flowScopeInstance, Optional.empty());
   }
+
+  public Optional<CatchEventTuple> findEscalationCatchEvent(
```

Comment:
🔧 It would be nice if we add some javadoc to this method as well. You could make it similar to the `findErrorCatchEvent` method above this one 📜🖊️